TypeProvider/Core/Naming.fs

```diff
@@ -0,0 +1,30 @@
+﻿[<RequireQualifiedAccess>]
+module internal Froto.TypeProvider.Core.Naming
```

Comment:
I have in mind to move this functionality into the Parser as an optional post-process pass, after processing includes and validating the AST to ensure all types used are defined somewhere.